package.json

```diff
@@ -33,8 +33,14 @@
     "lunr-languages": "^1.4.0"
   },
   "devDependencies": {
+    "@antora/site-generator-default": "^3.0.0-alpha.9",
     "chai": "^4.1.2",
     "dirty-chai": "^2.0.1",
+    "eslint": "^7.32.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.24.2",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
```

Comment:
I think we should not add a linter as part of this pull request. I'm in favor of adding a linter (eslint or standard) but once the migration is done.

Reply:
Agreed. And when we do add it, we should align with the linter configuration used in Antora core (which is open for refinement, of course).

Reply:
Sounds good, @Mogztter. I'll remove it from the PR. :+1:

@mojavelinux I believe the latest version is more (or less?) aligned after the previous update to the PR.

Reply:
Indeed. I was just mentioning it to clearly state the goal.